### PR TITLE
Create NEWS.md

### PR DESCRIPTION
**Pull request by gowthamrao**
https://github.com/OHDSI/ROhdsiWebApi/issues/47

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	Ability to post definitions https://github.com/OHDSI/ROhdsiWebApi/issues/42
+	Use checkmate for informative error message https://github.com/OHDSI/ROhdsiWebApi/issues/56
+	Deprecated functions not following design pattern https://github.com/OHDSI/ROhdsiWebApi/issues/66
+	Estimation specification https://github.com/OHDSI/ROhdsiWebApi/issues/69
+	Retrieve a list of definition metadata https://github.com/OHDSI/ROhdsiWebApi/issues/40
+	Delete definitions https://github.com/OHDSI/ROhdsiWebApi/issues/38
+5. New ROxygen support https://github.com/OHDSI/ROhdsiWebApi/issues/44
+	
+	
+ROhdsiWebApi 0.0.3
```
Comment (schuemie) [inline NEWS.md:20]: No point in adding old releases if we're not going to document the changes in them. Please remove.

File: NEWS.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+ROhdsiWebApi 0.1.0
+=====================
+Note: this version has breaking changes because design changes and naming conventions.
+
+1. Generic design pattern: Based on discussions here https://github.com/OHDSI/ROhdsiWebApi/issues/37
```
Comment (schuemie) [inline NEWS.md:5]: In general our OHDSI packages have two sections in NEWS (in line with CRAN expectations): Changes and Bug Fixes.

File: NEWS.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+2. Consistent naming conventions: https://github.com/OHDSI/ROhdsiWebApi/issues/86 Use of WebApi instead of Atlas https://github.com/OHDSI/ROhdsiWebApi/issues/21 and case https://github.com/OHDSI/ROhdsiWebApi/issues/22
+3. Created Vignette
+4. New functions: 
+	Incidence rate https://github.com/OHDSI/ROhdsiWebApi/issues/59
+	Patient profile https://github.com/OHDSI/ROhdsiWebApi/issues/57
+	Ability to post definitions https://github.com/OHDSI/ROhdsiWebApi/issues/42
+	Use checkmate for informative error message https://github.com/OHDSI/ROhdsiWebApi/issues/56
+	Deprecated functions not following design pattern https://github.com/OHDSI/ROhdsiWebApi/issues/66
```
Comment (schuemie) [inline NEWS.md:13]: Not a new function. Should be in a separate numbered item

File: NEWS.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+	Estimation specification https://github.com/OHDSI/ROhdsiWebApi/issues/69
+	Retrieve a list of definition metadata https://github.com/OHDSI/ROhdsiWebApi/issues/40
+	Delete definitions https://github.com/OHDSI/ROhdsiWebApi/issues/38
+5. New ROxygen support https://github.com/OHDSI/ROhdsiWebApi/issues/44
```
Comment (schuemie) [inline NEWS.md:17]: I don't think this is relevant for package users. Please delete

**Comment by schuemie** [comment]
General comments: 

- Please use the two sections (Changes and Bug Fixes)

- My personal preference is not to have full URLs when linking to issues, but rather use hyperlinked words / numbers (can be added using markdown syntax)

- The news is very cryptic, and currently would require a user to click on every link you provide and then read a complex issue thread to understand. The news should be stand-alone. For example, instead of saying 'Generic design pattern', briefly explain what the design pattern is in a few sentences). Another example: instead of 'New functions: Incidence rates', provide the names of the new functions.

File: NEWS.md (hunk 1 above)
Comment (gowthamrao) [reply NEWS.md:20]: Should we consider v0.1.0 as the official first release? If yes, that would change the way we document News.Rd and we can start over.

i.e. there would be no 'changes' as this is the first release and we can start clean

Comment (gowthamrao) [reply NEWS.md:20]: > Should we consider v0.1.0 as the official first release? If yes, that would change the way we document News.Rd and we can start over.
> 
> i.e. there would be no 'changes' as this is the first release and we can start clean

**Pull request merged**